### DISCUSSION TITLE
6 小時生成致命毒劑：千萬不能遭濫用的 AI

### DISCUSSION THREAD

**Discussion by Yao900108** [1101-GS4719-C/Python-Community-GS4719-C#553] [科技新知]
不少人都覺得製毒並不容易，至少需有化學知識。但如果請人工智慧幫忙呢？且 AI 系統「做」出的不是毒品而是毒藥。
公司Collaborations Pharmaceuticals 決定探索如何使用 AI 設計有毒分子，之前設計名為 MegaSyn 的商業 de novo 分子生成模型，藉助機器學習模型預測生物活性，尋找人類疾病標靶新治療抑製劑。
這種生成模型通常會懲罰預測毒性並獎勵預測目標活動，之後調整指導模型同時獎勵毒性和生物活性，並使用公共資料庫分子訓練人工智慧。調整後底層生成軟體建立在易獲得的開源軟體上，為了縮小分子範圍，將生成模型推向諸如神經毒劑 VX 類化合物。
VX 是神經毒劑的人造化學戰劑（化學戰劑：戰爭目的、有劇烈毒性、大規模毒害或殺傷敵方人畜和植物等化學物質），毒性強且作用迅速，6~10 毫克 VX 顆粒便足以致命。伺服器啟動後 6 小時內，新模型就生成 4 萬個分子。人工智慧不僅設計出 VX，還設計許多已知化學戰劑，更設計出許多看起來同樣合理的新分子，據預測值看，新分子比已知化學戰劑毒性更強。
https://technews.tw/2022/04/03/collaborations-pharmaceuticals/

**Comment by ThomasMeng0515** [comment]
或許在研究用途上有非常大的用處，但是如果落入不肖人士手中，後果可能不堪設想

**Comment by hapiwangy** [comment]
又是一種AI聰明過頭，但可能會危害到人類的例子。由此可知在AI的使用上要更小心一點，否則可能造成無法挽回的傷害

**Comment by ghost** [comment]
感覺發展上也沒回頭路了，只能看看之後是否有相關規範之類東西存在了

**Comment by HuanYuuuuuu** [comment]
隨著AI的發展越來越進步，其涉及到的領域也越來越廣
也應該要有相關規範與時俱進才能保證AI不會被濫用

**Comment by sean1325418** [comment]
科技的好壞原本就是取決在使用者的手裡

**Comment by PoooYen** [comment]
聰明反被聰明誤.....

**Comment by yenwei12138** [comment]
又是一個AI發展出的雙面刃，希望未來能有相關規範管理這種技術。

**Comment by Sazmafuji** [comment]
AI之後真不知會變何樣

**Comment by yi-1111** [comment]
規範使用才是未來必須面對的課題

**Comment by d26488399** [comment]
AI是一項工具，好或壞取決於使用者

**Comment by Yuhsin000** [comment]
又多一個要擔心AI的地方了

**Comment by Ku-Hao-Chen** [comment]
謝謝分享，希望AI不要造成太大的困難。

**Comment by Rosancl** [comment]
隨著AI不斷進步，AI不會只帶來對人類有益的，也會有諸如此類有害的一面。

**Comment by yuhao112** [comment]
科技越發展，需要更多的規範才能好好控制呢

**Comment by handsomeniko** [comment]
AI與倫理是很重要的議題

**Comment by henrywen547** [comment]
隨著AI越來越聰明，法律上的規範也要越來越多且更明確才行呢

**Comment by bloodtrail0511** [comment]
如果被不法分子拿走，不敢想像後果有多恐怖

**Comment by cordydeng** [comment]
不得不說雖然AI是必要的存在，但像這種危險的技術還是不要出現比較好

**Comment by jcpan821** [comment]
水能載舟，亦能覆舟

**Comment by 110409538** [comment]
科技是一把雙刃劍，就比如安眠藥可以讓失眠者度過夜晚，但吃多了就會中毒。

**Comment by minniewei** [comment]
我們永遠不知道ai持續的發展會造成甚麼

**Comment by Jay-722** [comment]
就看是被拿來做甚麼利用了

**Comment by liangyinglly** [comment]
應該讓ai研發出能自動解毒的方法

**Comment by HaoFish123** [comment]
感謝分享，真的要好好設置AI相關規範。

**Comment by Jimmy911218** [comment]
使用ai處理事情是每一個人的權利，但如何運用它是每一個人的道德

**Comment by treeann** [comment]
AI真的有好有壞

**Comment by stan890425** [comment]
像藥物合成都有許多困難點，一不小心就會變成毒藥

**Comment by alexthebigpp** [comment]
AI真的很強大，全看人怎麼使用他，壞的不是AI，是人。

**Comment by etin2468** [comment]
AI的使用範圍太廣了，被有心人士濫用不管在什麼領域都很危險
